packages/p2p/src/components/my-profile/my-profile.jsx

```diff
@@ -1,9 +1,95 @@
-import React from 'react';
-import './my-profile.scss';
+import React        from 'react';
+import PropTypes    from 'prop-types';
 import { localize } from 'deriv-translations';
+import                   './my-profile.scss';
+
+// TODO: This is just mockup data. it will be removed after Adding BE API.
+const user_data = {
+    firstName              : 'johny',
+    lastName               : 'bravo',
+    securityDeposit        : 0,
+    buyTrades              : 23,
+    sellTrades             : 25,
+    pastMonthTrades        : 42,
+    pastMonthCompletionRate: 99,
+    averageReleaseTime     : 5,
+    averageReleaseUnit     : 'min',
+};
+
+function capitalize (string) {
+    if (!string) {
+        return '';
+    }
+    return string[0].toUpperCase() + string.slice(1);
+}
+
+const Cell = (props) => {
+    return (
+        <div className='my-profile__cell'>
+            <div className='my-profile__cell-data'>
+                <div className='my-profile__cell-data-main'>
+                    {props.value}
+                    <sub className="my-profile__cell-data-dimension" >
+                        {props.dimension}
+                    </sub>
+                </div>
+                <div className='my-profile__cell-data-sub'>
+                    <div className='my-profile__cell-data-sub-upper'>
+                        {props.upperSubValue}
+                    </div>
+                    <div className='my-profile__cell-data-sub-lower'>
+                        {props.lowerSubValue}
+                    </div>
+                </div>
+            </div>
+            <div className='my-profile__cell-title'>
+                {props.title}
+            </div>
+        </div>
+    );
+};
+
+Cell.propTypes = {
+    title        : PropTypes.string,
+    value        : PropTypes.string,
+    dimension    : PropTypes.string,
+    upperSubValue: PropTypes.string,
+    lowerSubValue: PropTypes.string,
+};
 
 const MyProfile = () => {
-    return <div>{localize('MyProfile')}</div>;
+    return (
+        <div className='my-profile'>
+            <div className='my-profile__name'>
+                {`${capitalize(user_data.firstName)} ${capitalize(user_data.lastName)}`}
+            </div>
+            <div className='my-profile__data'>
+                <Cell
+                    title={localize('Security Deposit')}
+                    value={user_data.securityDeposit}
+                />
+                <Cell
+                    title={localize('Trades')}
+                    value={user_data.buyTrades + user_data.sellTrades}
+                    upperSubValue={localize('{{ buyTrades }} buy', { buyTrades: user_data.buyTrades })}
```

Comment:
we shouldn't have extra space in placeholders
```suggestion
                    upperSubValue={localize('{{buyTrades}} buy', { buyTrades: user_data.buyTrades })}
```